test/mocks/MockERC20.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
A similar file is avail. in the lib:

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/b0cf6fbb7a70f31527f36579ad644e1cf12fdf4e/contracts/mocks/ERC20Mock.sol

.

can be imported with: `import {ERC20Mock} from "@openzeppelin/contracts/mocks/token/ERC20Mock.sol";`

(either way is fine for me, just FYI)

Reply:
let's use libs whenever possible! It's one of the leftovers of the ai code

Reply:
ok ill change this

Reply:
turns out it is not even used anymore so i removed it